cardano-node/src/Cardano/Node/Startup.hs

```diff
@@ -97,11 +97,6 @@ data StartupTrace blk =
   -- | Warn when 'EnableP2P' is set.
   | P2PWarning
 
-  -- | Warn that peer-to-peer requires
```

Comment:
Is this deprecated? What's the story here

Reply:
P2P does not require to experimental protocols.  To set P2P it is enough to set `EnableP2P: True` (one does not need `TestEnableDevelopmentNetworkProtocols: True`)